features/example_groups/shared_examples.feature

```diff
@@ -25,8 +25,8 @@ Feature: shared examples
 
   ```ruby
   RSpec.shared_examples "some example" do |parameter|
-    \# Same behavior is triggered also with either `def something; 'some value'; end`
-    \# or `define_method(:something) { 'some value' }`
+    # Same behavior is triggered also with either `def something; 'some value'; end`
```

Comment:
Related issue
https://github.com/cucumber/common/issues/1413
https://github.com/cucumber/common/issues/1122

Is there a way around this?